and-cli.js

```diff
@@ -49,28 +51,10 @@ fixArgumentPosixPathConversion();
 
 program.parse(process.argv);
 
-// -----------------------------------------------------------------------------------------
-// #region Validation
-// -----------------------------------------------------------------------------------------
-
-// Map out the individual 'command' strings for comparison with parsed args
-// to see if invalid commands were passed in to the base program
-const commandStrings = commandObjects.map((commandObj) => commandObj.command);
-
-// Test to see if ALL args passed in are invalid. At least one of them should match a command here,
-// with the assumption that any additional mismatching args are intentionally being passed to a sub-command.
-let allParsedArgsInvalid = true;
-
-program.args.forEach((arg) => {
```

Comment:
Looks like we get this behavior out of the box now - I think this was originally put in to ensure that typing `and-cli migration -a SomeNewMigration` was valid, whereas `and-cli some new migration` was not. Seems to catch that and output an error for the user automagically now.